ng_server/src/main.rs

```diff
@@ -212,6 +214,13 @@ async fn get_target_nonce(Extension(state): Extension<Arc<State>>) -> Result<Str
     Ok(String::default())
 }
 
+async fn get_guess_probabalities(Extension(state): Extension<Arc<State>>) -> Result<Json<Vec<(String, f32)>>, Error> {
```

Comment:
little spelling typo:
```suggestion
async fn get_guess_probabilities(Extension(state): Extension<Arc<State>>) -> Result<Json<Vec<(String, f32)>>, Error> {
```